packages/venia-concept/src/components/Icon/icon.js

```diff
@@ -14,10 +55,16 @@ class Icon extends Component {
 
     render() {
         const { attrs, classes, name } = this.props;
-        const svg = Feather.icons[name].toSvg(attrs);
-        const fn = () => ({ __html: svg });
-
-        return <span className={classes.root} dangerouslySetInnerHTML={fn()} />;
+        // Use the name provided so that in the case of an unknown icon we throw
+        // an appropriate error.
+        const IconComponent = Icons[name] || name;
+        return (
+            <span className={classes.root}>
+                <IconComponent
+                    size={attrs && attrs.width}
+                    color={attrs && attrs.color} />
+            </span>
+        );
```

Comment:
The above is really the bulk of this change (aside from the stats addition). Does it make sense to dynamically import like so or should we put the onus of icon import/utilization on the components and do away with this abstraction? Aside from having all the icon imports in one place this component does provide a default styling.

Reply:
For example, per @DrewML: 
```js
// Inside MyComponent.js
import { SomeIcon } from 'react-feather';
import Icon from '../Icon';

<Icon src={SomeIcon} />
```

Reply:
I'm going to go ahead and do this ^